tests/tasks/create_test_interfaces.yml

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install dnsmasq
+  package:
+    name: dnsmasq
+    state: present
+
+
+- name: Create test interfaces
+  shell: |
+    # Manage all veth devices starting with test*
+    echo 'ENV{ID_NET_DRIVER}=="veth", ENV{INTERFACE}=="test*", ENV{NM_UNMANAGED}="0"' >/etc/udev/rules.d/88-veth.rules
+    udevadm control --reload-rules
+    udevadm control --reload-rules
+
+    # Create a namespace for devices
+    ip netns add testsetup
+
+    # Setup simulated device test1 and test2 with IPv6 auto support
+    ip link add test1 type veth peer name test1p
+    ip link set test1p netns testsetup
+    ip netns exec testsetup ip link set test1p up
+    ip link add test2 type veth peer name test2p
+    ip link set test2p netns testsetup
+    ip netns exec testsetup ip link set test2p up
+
+    # Create the 'testbr' - providing both 10.x ipv4 and 2620:52:0 ipv6 dhcp
```

Comment:
Please use RFC5737 addresses: https://tools.ietf.org/html/rfc5737

Reply:
ok, 192.0.2.1 and 2001:DB8::1 now used